### PR TITLE
expression_weighted visualize and subs

### PR DESCRIPTION
**Pull request by Yawgmoth90**

### CODE REVIEW

File: dcgpy/_subs_weights.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
+def _subs_weights(self, expr_str):
```
Comment (darioizzo) [inline dcgpy/_subs_weights.py]: I do not understand why expr_str is needed. I was expecting this to be ex. subs_weights() then simpy would be called internally to substitute the weights symbols with the weights values and return the symbolic expression with the substituted weights

Comment (darioizzo) [inline dcgpy/_subs_weights.py]: but the symbols for the weights are by convention w_i,j

Comment (Yawgmoth90) [inline dcgpy/_subs_weights.py]: I meant the symbols for the inputs!

Comment (darioizzo) [inline dcgpy/_subs_weights.py]: Then should this not be ex.simplify(["x", "y", "z"], subs_weights = True) ?

**Pull request merged**